index.js

```diff
@@ -0,0 +1,7 @@
+window.addEventListener('DOMContentLoaded', () => {
+  const btn = document.querySelector('#menu_btn');
+const menu = document.querySelector('#mobile-nav');
```

Comment:
- [ ] When the user clicks on any of the mobile menu options, a correct part of the page should be displayed, And the mobile menu is supposed to disappear. Here you are missing the second requirement, Please add an event to menu options so it should disappear when any of the mobile menu option is clicked.